addons/test_mail_full/tests/test_sms_composer.py

```diff
@@ -656,3 +656,20 @@ def test_composer_template_context_action(self):
             test_record_2.customer_id, None,
             content="Hello %s ceci est en français." % test_record_2.display_name
         )
+
+    def test_composer_sending_with_no_number_field(self):
+        test_record = self.env['mail.test.sms.partner'].create({'name': 'Test'})
+        sms_composer = self.env['sms.composer'].create({
+            'body': self._test_body,
+            'composition_mode': 'comment',
+            'mass_force_send': False,
+            'mass_keep_log': True,
```

Comment:
Can't we remove some of those values, as they are default/unused ? To keep the test as small as possible :)

Reply:
Done Changes